app/controllers/form_controller.rb

```diff
@@ -60,6 +60,20 @@ def review
           render_not_found
         end
       end
+      define_method("new_log_#{page.id}") do |_errors = {}|
```

Comment:
Doesn't need to be done as part of this PR but worth considering whether this needs to be a dynamically defined set of methods at all (think @jamesrose pointed this out previously) - currently it pretty much does the same thing for every page it's defined for anyway so could just be a single method they all route to with page id as a param.